example_cases/SPARC_PRD/input.yaml

```diff
@@ -4,21 +4,36 @@ algorithms:
   # and temperature, which defines a stored energy stored_energy. From this,
   # we can compute P_in from stored_energy and a tau_e scaling.
   - read_atomic_data
-  - calc_geometry
+  - calc_separatrix_elongation_from_areal_elongation
+  - calc_separatrix_triangularity_from_triangularity95
+  - calc_minor_radius_from_inverse_aspect_ratio
+  - calc_vertical_minor_radius_from_elongation_and_minor_radius
+  - calc_plasma_volume
+  - calc_plasma_surface_area
+  - calc_f_shaping_for_qstar
   - calc_q_star_from_plasma_current
-  - calc_fuel_average_mass_number
-  - calc_average_ion_temp
-  - calc_zeff_and_dilution_from_impurities
+  - calc_average_fuel_ion_mass
+  - calc_average_ion_temp_from_temperature_ratio
+  - calc_zeff_and_dilution_due_to_impurities
   - calc_plasma_stored_energy
-  - calc_power_balance_from_tau_e
+  - read_confinement_scalings
+  - solve_tau_e_scaling_for_input_power
   # Once we have P_in (=P_loss in steady-state), we want to split
   # it into components. For this, we need to estimate 1D plasma
   # profiles in the confined region.
-  - calc_beta
```

Comment:
So these grouped algorithms are now gone completely?

Reply:
Mostly yes. It's still possible to make algorithms of other algorithms, for example `cfspopcon.formulas.plasma_profiles.calc_peaked_profiles` which calculates the peaking before calculating the profile shape. You'd almost never call these algorithms separately, so I've grouped them for convenience, but there's still the option to call them separately if you'd like.